rest/taskana-rest-spring-example-wildfly/src/main/java/pro/taskana/wildfly/security/LoginErrorController.java

```diff
@@ -3,17 +3,19 @@
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.servlet.view.RedirectView;
 
 /** The loginError controller. */
 @Controller
 public class LoginErrorController {
 
-  @PostMapping
-  @GetMapping
-  @RequestMapping("/loginerror")
-  public RedirectView loginError() {
+  @GetMapping(path = "/loginerror")
+  public RedirectView loginErrorGet() {
```

Comment:
This is not very nice. Please use @Requestmapping(method = { GET, POST}) instead of duplicating the method. Thanks!

Reply:
This won't fix sonarcloud's issue: "This rule raises an issue when method is missing and when the method parameter is configured with more than one verb"